storybook/package.json

```
@@ -9,7 +9,7 @@
     "typecheck": "tsc -p ./tsconfig.json --noEmit"
   },
   "dependencies": {
-    "@elastic/charts": "^30.2.0",
+    "@elastic/charts": "latest",
```

Do you think a wildcard is more sensible? Your choice.

```suggestion
    "@elastic/charts": "*",
```

---

I've reverted that change for now

---

I think we move to `DEVELOPMENT` or something.